test/utils.test.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+beforeEach(() => jest.resetModules());
+
+test('bundleFeeds() throws error when bundling JS', async () => {
+    expect.hasAssertions();
+    jest.doMock('@asset-pipe/js-reader', () => async () => {
```

Comment:
`Promise.reject` is clearer in this case, I think

Reply:
done